extensions/poetry/main.ts

```diff
@@ -0,0 +1,53 @@
+import { PhylumApi } from "phylum"
+import { parse } from "https://deno.land/std/flags/mod.ts"
+
+// Add packages to the lockfile, and upload it to Phylum for analysis.
+// Return `true` if the packages pass the project's thresholds.
+// Return `false` if the packages don't pass the thresholds, or the processing
+// job is not finished.
+async function poetryCheck(args: string[]) {
+    console.log("Updating package lock…")
```

Comment:
This language does not quite line up with Python or `poetry`. Maybe `Attempting to add a dependency ...`

Generically, changing `lock` to `lockfile` could work but it has the downside of not being precise here since _both_ the `pyproject.toml` and `poetry.lock` file are updated with a `poetry add` command.